sql/updates/world/master/2018_08_18_98_world.sql

```diff
@@ -0,0 +1,6 @@
+-- Demon Hunter SAI
```

Comment:
Can you explain me what's the point of this script ?

Reply:
It fixes https://www.wowhead.com/quest=39877/in-the-loop